src/pages/fundamentals/eutxo-patterns.mdx

```diff
@@ -16,11 +16,11 @@ pre code {
 # EUTxO Patterns and Practices
 
 There are number of design patterns that have been refined and enabled as Plutus has advanced from V1 to V2,
-and now to V3. This document seeks to be a non-exhaustive reference to these design patterns and practices.
+and now to V3. This document seeks to be a reference to these design patterns and practices.
```

Comment:
I believe that being non-exhaustive is likely to be the case as we, (the Cardano community), discover new design patterns we can add them to this document. Just my two cents.